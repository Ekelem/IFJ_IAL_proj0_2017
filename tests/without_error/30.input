

scope
	dim x as integer = 3
	dim a as double = 36.36
	do while x > (18 - 17*1 - 1)
		print a;
		x = x - 1
		a = a - 6.4
	loop
end scope