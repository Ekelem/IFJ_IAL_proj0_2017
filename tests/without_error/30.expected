 36.36 29.96 23.56